sirepo/job_driver/__init__.py

```diff
@@ -336,23 +330,23 @@ def _receive(self, msg):
             ('opName' not in c or c.opName == job.OP_ERROR)
             or ('reply' in c and c.reply.get('state') == job.ERROR)
         ):
-            pkdlog('error agentId={} msg={}', self._agentId, c)
+            pkdlog('self={} error msg={}', self, c)
         elif c.opName == job.OP_JOB_CMD_STDERR:
-            pkdlog('stderr from job_cmd agentId={} msg={}', self._agentId, c)
+            pkdlog('self={} stderr from job_cmd msg={}', self, c)
             return
         else:
-            pkdlog('opName={} agentId={} opId={}', c.opName, self._agentId, i)
+            pkdlog('self={} opName={} opId={:.6}', self, c.opName, i)
         if i:
             if 'reply' not in c:
-                pkdlog('agentId={} No reply={}', self._agentId, c)
+                pkdlog('self={} no reply={}', self, c)
                 c.reply = PKDict(state='error', error='no reply')
             if i in self.ops:
                 #SECURITY: only ops known to this driver can be replied to
                 self.ops[i].reply_put(c.reply)
             else:
                 pkdlog(
-                    'agentId={} not pending opId={} opName={}',
-                    self._agentId,
+                    'self={} not pending opId={:.6} opName={}',
```

Comment:
This is when an opId might be good to be its own object (subclass of str) so we can implement pkdebug_str for it. It's a bit of a pain, because serialization fails (I think), but we could possibly implement a pkjson serializer function that allows us to serialize anything. Maybe pkcollections does that already with str.

Reply:
Hmm I'm 50/50. We would have to hand deserialize the opId in the message to an opId object which is work and I'm not sure is worth the upside. We also don't do that anywhere else and it seems like something we may want to build a structure for (deserializing message payloads, or parts of them, into objects). Thoughts?